tensorboard/webapp/feature_flag/types.ts

```diff
@@ -20,4 +20,10 @@ export interface FeatureFlags {
   inColab: boolean;
   // Whether to enable our experimental GPU line chart.
   enableGpuChart: boolean;
+  // Maximum number of runs to include in a request to get scalar data.
+  // `undefined` indicates that we should rely on defaults defined in the
+  // dashboards code.
+  //
+  // See: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.ts
```

Comment:
You may also want to tag https://github.com/tensorflow/tensorboard/blob/648ffe001eb910e68493506cbe468f9caab0f9c6/tensorboard/webapp/metrics/effects/index.ts#L192-L201.

Reply:
I unfortunately don't really understand.  It doesn't look to me like there is anything related to batching in that code.

Reply:
Timeseries batches the request by default and it is not tunable today. This may prevent you from adopting TimeSeries but I just wanted to mention that this flag can be reused for timeseries.